chef/cookbooks/provisioner/templates/default/autoyast-upgrade.xml.erb

```diff
@@ -90,6 +90,20 @@
           ]]>
         </source>
       </script>
+<% if @needs_openvswitch -%>
+      <script>
+        <chrooted config:type="boolean">true</chrooted>
+        <debug config:type="boolean">true</debug>
+        <filename>openvswitch_upgrade</filename>
+        <source>
+          <![CDATA[
+          chkconfig -d openvswitch-switch
+          systemctl enable openvswitch
+          rm /etc/init.d/ovs-ifup-br-*
+          ]]>
+        </source>
+      </script>
+<% end -%>
```

Comment:
It actually runs after it, inside chroot (`chrooted` set to true, unlike the other script)